docs/content.md

```diff
@@ -517,28 +521,29 @@ navigation_title:
 navigation_description: 
 keywords: 
 
-## Preview a change
-1. Go to the terminal tab on Visual Studio Code **or** Go the top bar of your laptop - click terminal - click 'new terminal'
-2. A line of code will come up and you should type in **bin/dev**, this will start a series of code
-3. When it has finished running, go into the ports tab and click the globe icon by 127.0.0.1:3000. This will open the browser
-4. When putting in a page to preview it, right click the page title on the left hand side bar in Visual Studio Code and copy path
-5. Go to the url: http://127.0.0.1:3000 and insert a / on the end
-6. The copied path will be something like this: /workspaces/get-into-teaching-app/app/views/content/a-day-in-the-life-of-a-teacher.md
-7. Edit this path to only be the necessary page path: /a-day-in-the-life-of-a-teacher
-The url is http://127.0.0.1:3000/a-day-in-the-life-of-a-teacher
-8. You will need to CTL-Save a change in Visual Studio Code before viewing it in preview. You can just refresh the preview URL after you save a change to view it.
-
 ## Saving a change
 When you have made a change in Visual Studio Code and want to save your work.
 1. Save the change by CTRL-S
 2. This will cause a notification on the source control
 3. You will need to type a name for this change in the 'message' box
 4. Click the plus button that appears on the change - this will stage the change
 5. Click 'sync changes'
-6. Commit the changes
-7. This will then open a pull request, you will need to fill in the details for this and click 'Create pull request'
+6. Commit the changes - make sure you name each commit in the box above the blue box where it says commit
```

Comment:
Can we add something about making the commit name meaningful and distinct as it is really helpful when reviewing the PR / as an audit trail